modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/struts/BaseSamlStrutsAction.java

```diff
@@ -92,21 +92,14 @@ public String execute(
 		getSamlProviderConfigurationHelper();
 
 	public boolean isEnabled() {
-		return samlProviderConfigurationHelper.isEnabled();
-	}
-
-	public void setSamlProviderConfigurationHelper(
-		SamlProviderConfigurationHelper samlProviderConfigurationHelper) {
-
-		this.samlProviderConfigurationHelper = samlProviderConfigurationHelper;
+		return getSamlProviderConfigurationHelper().isEnabled();
```

Comment:
Hmm this passes SF..

Reply:
@W2jx Do not do this even if it passed SF.